docs/basic-features/image-optimization.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Next.js supports built-in image optimization as well as 3rd party loaders for Imgix, Cloudinary, and more! Learn more here.
+---
+
+# Image Optimization
+
+<details open>
+  <summary><b>Examples</b></summary>
+  <ul>
+    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/basic-image-optimization">Basic Image Optimization</a></li>
+  </ul>
+</details>
+
+Since version **10.0.0**, Next.js provides an Image component that will automatically optimize the source image based on the browser. The optimization includes resizing the image based on the browser's viewport as well as selecting the best format, such as [WebP](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types), based on the browser's [`Accept`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) header.
+
+The objective of the Image component is to improve performance of Next.js applications, with a particular focus on improving their Core Web Vitals scores.
+
+## Default Behavior
+
+To add an image to your application, import the Image component on your [page](./pages).
+
+```jsx
+import Image from 'next/image'
+
+function Home() {
+  return (
+    <>
+      <h1>My Homepage</h1>
+      <Image src="/me.png" width={200} height={200} />
+      <p>Welcome to my homepage!</p>
+    </>
+  )
+}
+
+export default Home
+```
+
+## Configuration
+
+If you want to configure Image Optimization you can do so by using `images` in `next.config.js`.
+
+### Sizes
+
+You can specify which size widths you wish to allow using the `sizes` property. Since images will maintain aspect ratio using the `width` and `height` of the source image, there is no need to specify height, only the width. You can think of these as breakpoints.
```

Comment:
```suggestion
You can specify a list of image widths to allow using the `sizes` property. Since images maintain their aspect ratio using the `width` and `height` attributes of the source image, there is no need to specify height in `next.config.js` – only the width. You can think of these as breakpoints.
```